tests/functional/pointerevent_capture_suppressing_mouse-manual.js

```diff
@@ -0,0 +1,30 @@
+define(function(require) {
+	var registerSuite = require('intern!object');
+	var w3cTest = require('../support/w3cTest');
+	var name = 'pointerevent_capture_suppressing_mouse-manual';
+
+	registerSuite({
+		name: name,
+
+		main: function() {
+			return w3cTest(this.remote, name + '.html')
+				.findById('target0')
+					.moveMouseTo(50, 50)
+					.clickMouseButton(0)
+					.end()
+				.findById('target1')
+					.moveMouseTo(50, 30)
+					.clickMouseButton(0)
+					.end()
+				.findById('btnCapture')
+					.moveMouseTo(50, 4)
+					.pressMouseButton(0)
+					.end()
+				.findByTagName('body')
```

Comment:
Well, the assertion is that you don't get events fired on the targets because capture is enabled. If you don't hover over the targets, then of course the events won't fire. It's possible that the test isn't defensive enough to ensure that all the necessary steps are taken to prevent false positives.